src/leases/components/leaseSections/rent/ContractRentEdit.tsx

```diff
@@ -210,6 +245,9 @@ class ContractRentEdit extends PureComponent<Props> {
         <Column small={6} medium={4} large={2}>
           {intendedUseField}
         </Column>
+        { leaseServiceUnitId !== ServiceUnitIds.MAKE && <Column small={6} medium={4} large={2}>
```

Comment:
A Comment here could clarify why this is not shown for MAKE

Reply:
Added in the other PR in commit https://github.com/City-of-Helsinki/mvj-ui/pull/544/commits/494f4e63e087239105cb47bbd71561961bce3345